README.md

```diff
@@ -2,29 +2,84 @@
 
 Product focuses on 100% education as well as upskilling developing countries and rural areas. 
 
-## Available Scripts
+# How To Run the Project
 
-In the project directory, you can run:
+***1.*** Open Project Folder</br>
 
-### `yarn start`
+***2.*** To install all dependencies run
+```
+yarn install
+```
+***3.*** To run development server.This will run the development server in the http://localhost:3000
+```
+yarn start
+```
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+# To start contributing, follow the below guidelines: 
 
-### `yarn test`
+**1.**  Fork [this](https://github.com/codeforcauseorg/edu-client.git) repository.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+**2.**  Clone your forked copy of the project.
 
-### `yarn build`
+```
+git clone -b <branch_name> https://github.com/<your_user_name>/edu-client.git
+```
```

Comment:
```diff
- git clone -b <branch_name> https://github.com/<your_user_name>/edu-client.git
+ git clone https://github.com/<your_user_name>/edu-client.git
```

You are either way mentioning to create a new branch on step 7.

Reply:
Yes I am on it. Btw the project already contains the yarn.lock file and on installing dependencies with npm would introduce package-lock.json file. We get a warning to use one package manager in one project. But still if you want , I can go ahead and add the commands for it as well.